### DISCUSSION TITLE
Separate files storage folder by data model

### DISCUSSION THREAD

**Discussion by rungsan** [directus/directus#15159] [Archive]
Is it possible to define storage folder for each data model?

**Comment by JonathanSchndr** [comment]
> Is it possible to define storage folder for each data model?

Folders are possible as a virtual structure for the organization: https://docs.directus.io/reference/system/folders.html
But this structure is only virtual and will not be transferred to the file system, because the files are all ordered by a scheme and are only classified with meta data.

You opened the thread as Idea, but since you ended the sentence with a question mark, I assume it was a question?

**Comment by rijkvanzanten** [comment]
What do you mean by "each data model"? Do you mean per collection?
